aidants_connect_habilitation/models.py

```diff
@@ -66,6 +66,8 @@ def save(
     ):
         if self.email:
             self.email = self.email.lower()
+        if update_fields is not None:
+            update_fields = {"email"}.union(update_fields)
```

Comment:
https://docs.djangoproject.com/en/4.2/releases/4.2/#setting-update-fields-in-model-save-may-now-be-required